test/tools/OpenCover/OpenCover.psm1

```diff
@@ -332,18 +304,19 @@ function Install-OpenCover
         }
         else 
         {
-            throw "$TargetDirectory/OpenCover exists"
+            throw "$TargetDirectory/OpenCover exists, not continuing.  Use -force to re-install"
         }
     }
 
-    $webclient.DownloadFile($packageUrl, "$env:TEMP/$filename")
-    if ( ! (test-path $env:TEMP/$Filename) )
+    Invoke-WebRequest -Uri $packageUrl -OutFile "$tempPath"
+    if ( ! (test-path $tempPath) )
     {
         throw "Download failed: $packageUrl"
     }
     
     import-module Microsoft.PowerShell.Archive
-    Expand-ZipArchive -Path "$env:TEMP/$filename" -DestinationPath "$TargetDirectory/OpenCover"
+    Expand-Archive -Path $tempPath -DestinationPath "$TargetDirectory/OpenCover"
```

Comment:
>Expand-Archive [](start = 4, length = 14)

This should be Expand-ZipArchive as Expand-Archive is not available on PSv4. Azure functions does not support PSv5 yet.